zebra-state/src/request.rs

```diff
@@ -108,10 +112,19 @@ impl From<Arc<Block>> for FinalizedBlock {
             .coinbase_height()
             .expect("finalized blocks must have a valid coinbase height");
         let hash = block.hash();
+        let transaction_hashes = block
+            .transactions
+            .iter()
+            .map(|tx| tx.hash())
+            .collect::<Vec<_>>();
```

Comment:
We're doing this pattern a lot

Reply:
Yes, there are various places where we compute transaction hashes.